Source/ContextLabel.swift

```diff
@@ -135,18 +135,18 @@ open class ContextLabel: UILabel, NSLayoutManagerDelegate, UIGestureRecognizerDe
   }
   
   public var underlineStyle: (LinkResult) -> NSUnderlineStyle = { _ in
-    return .styleNone
+    return []
   }
   
   public var modifiedAttributedString: ((NSAttributedString) -> NSAttributedString)?
   
   public var didTouch: (TouchResult) -> Void = { _ in }
   
-  public var didCopy: (String!) -> Void = { _ in }
+  public var didCopy: (String?) -> Void = { _ in }
```

Comment:
@michaelloistl please check this change. I think it is okay, but it should be reviewed. See https://swift.org/blog/iuo/ for more info

Reply:
Keeping the optional value as per your PR. Have also changed the delegate version of it.

Reply:
Ok!